### ISSUE TITLE
There is no variable expansion for jinja2 in the summary page of ansible-navigator

### ISSUE THREAD

**Issue by hiyokotaisa** [ansible/ansible-navigator#1096]
##### ISSUE TYPE

- Bug Report

##### SUMMARY

When using Variable as the task name like `Check status of {{ package }} service`, ansible-navigator will not expand it.

##### ANSIBLE-NAVIGATOR VERSION

```
student13@ansible-1 ~]$ ansible-navigator --version
ansible-navigator 1.1.0
```

##### STEPS TO REPRODUCE

1. Create playbook with task name that has variable

```yaml
---
- name: Check Status
  hosts: node1
  become: true
  vars:
    package: "httpd"

  tasks:
    - name: Check status of {{ package }} service
      service_facts:
      register: service_state

    - debug:
        var: service_state.ansible_facts.services["{{ package }}.service"].state
```

3. Run it from ansible-navigator

##### EXPECTED RESULTS

I can see the actual value of variable in task name in summary page.

##### ACTUAL RESULTS

The variable does not expand.
![image1](https://user-images.githubusercontent.com/4246532/158789709-f6ccf6ba-c373-4235-9ed4-faccc29b4eae.png)

However, I can see the variable expands in task detail page.
![image2](https://user-images.githubusercontent.com/4246532/158790107-af1310bc-12a2-427c-afcf-3e208284c160.png)

**Comment by hiyokotaisa** [comment]
I can see the similar issue https://github.com/ansible/ansible-navigator/issues/541 but it seems the issue still exists in 1.1.0.

**Comment by ganeshrn** [comment]
The fix will be available in 2.0.0 release. For now you can verify if the issue is fixed in the main branch/
